### ISSUE TITLE
can't $ref to local definitions if the schema is referred from a file

### ISSUE THREAD

**Issue by polimediaupv** [jdorn/json-editor#367]
Have an schema with 
"definitions":{
           "something":{...}
}
in the root of the schema, if that schema is included as a javascript variable i can ref to #/definitions/something and it links ok but if i link the schema file using href
schema: {
          $ref: "nameofschema.json"
        }

The linkage does not work to me.

**Comment by stevage** [comment]
Yeah, I just ran across this. Loading the JSON file via AJAX as in the example:
```
          editor = new JSONEditor(document.getElementById('editor_holder'),{
            ajax: true,
            schema: { $ref: "schema2.json" },
```

failed. (Any `$ref` failed to resolve)

But manually loading the file first with AJAX and then doing:

```
            schema: schema
```

worked fine.

**Comment by mustangboy50** [comment]
Thanks for posting the solution. Manually loading the file worked for me.
